ftest.sh

```diff
@@ -88,41 +82,9 @@ cleanup() {
     i=5
     while [ $i -gt 0 ]; do
         if clush "${CLUSH_ARGS[@]}" -B -l "${REMOTE_ACCT:-jenkins}" -R ssh \
-             -S -w "$(IFS=','; echo "${nodes[*]}")" "set -x
-            if grep /mnt/daos /proc/mounts; then
-                if ! sudo umount /mnt/daos; then
-                    echo \"During shutdown, failed to unmount /mnt/daos.  \"\
-                         \"Continuing...\"
-                fi
-            fi
-            x=0
-            if grep \"# DAOS_BASE # added by ftest.sh\" /etc/fstab; then
-                nfs_mount=true
-            else
-                nfs_mount=false
-            fi
-            sudo sed -i -e \"/added by ftest.sh/d\" /etc/fstab
-            if [ -n \"$DAOS_BASE\" ] && \$nfs_mount; then
-                while [ \$x -lt 30 ] &&
-                      grep $DAOS_BASE /proc/mounts &&
-                      ! sudo umount $DAOS_BASE; do
-                    sleep 1
-                    let x+=1
-                done
-                if grep $DAOS_BASE /proc/mounts; then
-                    echo \"Failed to unmount $DAOS_BASE\"
-                    exit 1
-                fi
-                if [ -d $DAOS_BASE ] && ! sudo rmdir $DAOS_BASE; then
-                    echo \"Failed to remove $DAOS_BASE\"
-                    if [ -d $DAOS_BASE ]; then
-                        ls -l $DAOS_BASE
-                    else
-                        echo \"because it does not exist\"
-                    fi
-                    exit 1
-                fi
-            fi"; then
+             -S -w "$(IFS=','; echo "${nodes[*]}")"                        \
+             "DAOS_BASE=$DAOS_BASE
+             $(sed -e '1,/^$/d' "$PREFIX"/lib/daos/TESTING/ftest/scripts/cleanup_nodes.sh)"; then
```

Comment:
(style)  line over 80 characters

Reply:
Can't be reasonably broken due to long strings and indentation.